### PR TITLE
Update viewer.html for Internet Explorer Compatibility

### PR DESCRIPTION
**Pull request by WilliamRClark**
If your Internet Explorer 11 default compatibility settings are set to "I.E 7 Compatibility", the PDF plugin will not load.  This fix is the same one used by AngularJS to force the browser to use HTML5 mode.

### CODE REVIEW

**Comment by timvandermeij** [comment]
@WilliamRClark After addressing the comment above, please squash the resulting commits into one using https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.

**Comment by yurydelendik** [comment]
> Change this to  to fix a typo.
@timvandermeij I think we can remove this comment.

But I think we enclose the meta inside `<!--#if GENERIC -->` to avoid polluting for Firefox and Chrome.

**Comment by WilliamRClark** [comment]
Comment changes committed.
 


     On Monday, August 31, 2015 7:39 PM, Yury Delendik <notifications@github.com> wrote:
   

 
Change this to  to fix a typo.
@timvandermeij I think we can remove this comment.
But I think we enclose the meta inside  to avoid polluting for Firefox and Chrome.—

File: web/viewer.html [hunk 1]
```diff
@@ -27,6 +27,8 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+    <!-- Ignore legacy IE compatibility settings and prefer HTML5 -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (Snuffleupagus) [inline web/viewer.html]: Please also address the review comments in https://github.com/mozilla/pdf.js/pull/6374#issuecomment-136528408, i.e. change the patch to:
```html
<!--#if GENERIC-->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
<!--#endif-->
```
After this, please squash the commits, see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.

Comment (WilliamRClark) [inline web/viewer.html]: Still have to validate with Edge, the other versions of Internet Explorer.

Will commit this change when testing is complete.

Comment (WilliamRClark) [inline web/viewer.html]: I still need to test this change on Edge, and I.E. 9.0 and 10.0.  Will commit once this is done. 


     On Tuesday, September 1, 2015 8:10 AM, Jonas Jenwald <notifications@github.com> wrote:
   

 In web/viewer.html:> @@ -27,6 +27,8 @@
>  
>      <meta name="google" content="notranslate">
>  
> +    
> +    <meta http-equiv="X-UA-Compatible" content="IE=edge">
Please also address the review comments in #6374 (comment), i.e. change the patch to:
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
After this, please squash the commits, see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.—

File: web/viewer.html [hunk 2]
```diff
@@ -27,6 +27,8 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+    <!-- Ignore legacy IE compatibility settings and prefer HTML5 -->
```
Comment (Rob--W) [inline web/viewer.html]: I suggest to remove this comment. Everyone who is familiar with IE knows what the meta tag does. And otherwise it is just one internet search away.

**Comment by WilliamRClark** [comment]
For maximum de-cluttering:


<meta http-equiv="X-UA-Compatible" content="IE=edge">
<![endif]>

**Comment by Rob--W** [comment]
@WilliamRClark We don't have IE-specifc builds.

File: web/viewer.html [hunk 3]
```diff
@@ -27,6 +27,9 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+<!--#if GENERIC || CHROME-->
```
Comment (Snuffleupagus) [inline web/viewer.html]: This line should read `<!--#if GENERIC-->`, since this code is *not* needed in Chrome. Also, please remove the trailing whitespace at the end of this line.

Finally, you also need to squash the commits, please see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.

Comment (WilliamRClark) [inline web/viewer.html]: Getting rid of the Chrome reference.

**Comment by WilliamRClark** [comment]
All code review changes should be complete.

**Comment by timvandermeij** [comment]
@WilliamRClark Looks good. All that needs to be done is to squash the commits into one commit. See https://github.com/mozilla/pdf.js/wiki/Squashing-Commits on how to do that. We can land this patch after that has been done.

**Comment by yurydelendik** [comment]
Merged by b1515fdd1060e3a68776b9fe022cd830f66e5f7d.

Thank you for the patch.
